app/src/main/res/layout/screen_patients_v2.xml

```diff
@@ -64,11 +64,22 @@
         android:src="@drawable/illustrations_homescreen"
         tools:ignore="ContentDescription" />
 
-      <org.simple.clinic.patient.recent.RecentPatientsView
-        android:id="@+id/patients_record_recent_patients"
+      <LinearLayout
```

Comment:
We could continue using RelativeLayout itself without introducing LinearLayout here? Or you may wanna do this in next PR.

Reply:
This is just a temporary code until we do our integration.